src/shared-styles/_typography.scss

```diff
@@ -44,3 +54,13 @@
   font-weight: $font-weight;
   line-height: $_baseline-size / $_font-size;
 }
+
+@mixin _label-typography($color, $uppercase: true) {
+  color: $color;
+  $letterspacing: 0.5px !default;
```

Comment:
Remove the `!default` as we don't want this to be overridden. 

Rename to `$_letterspacing` as it's local.

Reply:
Or even better, as discussed, just set it directly and lose the variable :-)